modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -520,9 +520,7 @@ class ApiClient(object):
         for auth in auth_settings:
             auth_setting = self.configuration.auth_settings().get(auth)
             if auth_setting:
-                if not auth_setting['value']:
```

Comment:
This change breaks my client using openapi-generator-cli-5.0.0-20200127.021040-34.jar

We have api endpoints which allow Any Auth method. This function needs an overhaul, because it simply picks the last configured setting in the array, instead of the one which has actually been specified in the Configuration instance...
Preserving this line makes the code do the right thing...

    if not auth_setting['value']:
                    continue

Reply:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/unittest/mock.py", line 1183, in patched
    return func(*args, **keywargs)
  File "/home/circleci/app/snap_usertracking/tests/test_client_api.py", line 665, in test_post_apikey
    response = api_instance.user_event(correct_post_data)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/api/tracking_api.py", line 287, in user_event
    **kwargs)  # noqa: E501
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/api/tracking_api.py", line 402, in user_event_with_http_info
    collection_formats=collection_formats)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/api_client.py", line 378, in call_api
    _request_timeout, _host)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/api_client.py", line 190, in __call_api
    _request_timeout=_request_timeout)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/api_client.py", line 422, in request
    body=body)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/rest.py", line 313, in POST
    body=body)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/snap_usertracking_api/rest.py", line 170, in request
    headers=headers)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/urllib3/request.py", line 80, in request
    method, url, fields=fields, headers=headers, **urlopen_kw
  File "/home/circleci/app/venv/lib/python3.6/site-packages/urllib3/request.py", line 171, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/urllib3/poolmanager.py", line 330, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/circleci/app/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/home/circleci/app/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 387, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/local/lib/python3.6/http/client.py", line 1262, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/lib/python3.6/http/client.py", line 1303, in _send_request
    self.putheader(hdr, value)
  File "/usr/local/lib/python3.6/http/client.py", line 1239, in putheader
    if _is_illegal_header_value(values[i]):
TypeError: expected string or bytes-like object

Reply:
> We have api endpoints which allow Any Auth method. 

Can you clarify a bit what you mean by "Any" auth?

Reply:
ahh, I have a custom configuration.mustache but based on an older version.... I'll try updating it and see if that works...

Reply:
> ahh, I have a custom configuration.mustache but based on an older version.... I'll try updating it and see if that works...

I see. Let me know how it goes.